setup.py

```diff
@@ -131,8 +130,9 @@ def combine_files(*args):
     scripts=[
         'bin/tower-cli',
     ],
-    packages=find_packages(),
-
+    packages=[
+        'tower_cli',
+    ],
```

Comment:
I tried testing this by doing `sudo python setup.py sdist`, and I found that both docs and tests were still getting included.

Sidebar: if we do exclude tests, I would prefer to use the syntax of `find_packages(exclude=['tests'])`

If I remove tests from `MANIFEST.in`, then I find that it really does not include the folder.

However, I don't feel like we have universal agreement that the tests should not be included, see #201. So I think the best course of action for the moment would be to role back the change this this line. As of yet I don't think it's having an effect anyway.